v8env/test/cookie_jar.spec.js

```diff
@@ -0,0 +1,23 @@
+import { expect } from 'chai'
```

Comment:
This file is in the old test path, needs to be in `tests/v8env/tests/` to be ran